docs/quickstart-cli.md

```diff
@@ -121,8 +126,11 @@ kubectl apply -f https://raw.githubusercontent.com/rancher/elemental-docs/main/e
 
 ## Preparing the installation (seed) image
 
+
+This is the last step, you need an Elemental Teal seed image that includes automatically (not for aarch64 yet) the initial registration config, so it can be auto registered, installed and fully deployed as part of your cluster.
```

Comment:
```suggestion
This is the last step: You need an Elemental Teal seed image that includes automatically (not for aarch64 yet) the initial registration config, so it can be auto registered, installed and fully deployed as part of your cluster.
```

Reply:
What's that aarch64 remark about ?

Reply:
It seems to me the idea is that it is not `automatic` for aarch64 🤷🏼 
I am going to drop that remark and also remove *automatically*...